config/config.go

```diff
@@ -152,11 +152,7 @@ func FromFlags() (*Config, error) {
 		cfg.LogLevel = zerolog.ErrorLevel
 	}
 
-	if logWriter == "stderr" {
-		cfg.LogWriter = os.Stderr
-	} else {
-		cfg.LogWriter = zerolog.NewConsoleWriter()
-	}
+	cfg.LogWriter = zerolog.NewConsoleWriter()
```

Comment:
If we want to produce valid JSON log lines, then this won't work. What we need is:
```go
cfg.LogWriter = os.Stdout
```

This will produce log lines such as:
```json
{
    "level": "debug",
    "component": "API",
    "url": "/",
    "id": 1,
    "method": "eth_blockNumber",
    "params": [],
    "is-ws": false,
    "time": "2024-03-11T14:37:57+02:00",
    "message": "API request"
}
```

and
```json
{"message":"{\"level\":\"debug\",\"component\":\"API\",\"id\":7,\"hash\":\"0x559b1c7a3c8bf1383bad293cbbded98ab9b864df728cdc4bc5a4df4cb521c9ca\",\"number\":\"0x1\",\"parentHash\":\"0x716d10fd2eb72b01e876580d348d8a9069e4a3395ab7cc27196971f04e9280f4\",\"receiptsRoot\":\"0x0000000000000000000000000000000000000000000000000000000000000000\",\"transactions\":[\"0xe4416fdf495e56d3467ecc1e04055f7a1cb4fe2b4fa180d2f7cde472879aafa7\"],\"time\":\"2024-03-11T14:17:43+02:00\",\"message\":\"API response\"}"}
```